java-checks/src/main/java/org/sonar/java/checks/PrintfCheck.java

```diff
@@ -107,11 +109,15 @@ protected void onMethodInvocationFound(MethodInvocationTree mit) {
         return;
       }
       verifyParameters(mit, args, params);
-    } else if (formatStringTree.is(Tree.Kind.PLUS)) {
+    } else if (formatStringTree.is(Tree.Kind.PLUS) && operandsAreOnSameLine((BinaryExpressionTree) formatStringTree)) {
```

Comment:
With such implementation we will drop reported False Positives, but we won't be able to detect issue such as:
```java
String.format("value is %d"
      + " not replaced"); // False negative, "%d" is not replaced
```